GOVERNANCE.md

```diff
@@ -0,0 +1,182 @@
+
+# Project Governance
+
+<!-- TOC -->
+
+* [Triagers](#triagers)
+* [Collaborators](#collaborators)
+  * [Collaborator activities](#collaborator-activities)
+* [Technical steering committee](#technical-steering-committee)
+  * [TSC meetings](#tsc-meetings)
+* [Collaborator nominations](#collaborator-nominations)
+  * [Onboarding](#onboarding)
+* [Consensus seeking process](#consensus-seeking-process)
+
+<!-- /TOC -->
+
+## Triagers
+
+Triagers assess newly-opened issues in the Systelab libraries repositories. The GitHub team for Systelab libraries triagers is @systelab/angular-team.
+Triagers are given the "Triage" GitHub role and have:
+
+* Ability to label issues and pull requests
+* Ability to comment, close, and reopen issues and pull requests
+
+See:
+
+* [List of triagers](./README.md#triagers)
+* [A guide for triagers](./doc/guides/contributing/issues.md#triaging-a-bug-report)
+
+## Collaborators
+
+Systelab libraries core collaborators maintain the GitHub repository.
+The GitHub team for Systelab libraries core collaborators is @systelab/anguler-team.
+Collaborators have:
+
+* Commit access to the [systelab/systelab-components][] repository
```

Comment:
You refer to systelab-components.
Do you want to refer it or do you want a general document?